src/accessories/HandHelper.ttslua

```diff
@@ -37,6 +52,55 @@ function updateColors()
   self.setName(handColor .. " Hand Helper")
 end
 
+function getHandSizeModifier()
+  local modSize = 0
+  local deckAreaObjects = playermatApi.getDeckAreaObjects(matColor)
+
+  -- look for  cards around playermat that modify hand size
+  for _, obj in ipairs(playermatApi.searchAroundPlayermat(matColor, "isCard")) do
+    if not obj.is_face_down and
+      obj ~= deckAreaObjects.draw and
+      obj ~= deckAreaObjects.topCard and
+      obj ~= deckAreaObjects.discard then
+
+      -- handle specific cards
+      local md = JSON.decode(obj.getGMNotes()) or {}
+      if md.id == "60205" then -- Arcane Enlightenment
+        modSize = modSize + 1
+      elseif md.id == "02020" then -- Laboratory Assistant
+        modSize = modSize + 2
+      elseif md.id == "60202" then -- Vault of Knowledge
+        modSize = modSize + 2
+      elseif md.id == "03041" then -- Drawing the Sign
+        modSize = modSize - 5
+      elseif md.id == "71023" then -- Dr. Mya Badry
+        modSize = modSize + 2
+      end
+    end
+  end
+  return modSize
+end
+
+function getInvestigatorHandSize()
+  local activeInvestigatorData = playermatApi.getActiveInvestigatorData(matColor)
+  if activeInvestigatorData == nil then return 8 end
+  -- Patrice Hathaway has a base handsize of 5
+  if activeInvestigatorData.id == "06005" then
+    return 5
+  elseif activeInvestigatorData.id == "11017" then
+    -- if George Barnaby, look for attachment helper nearby set to his state
+    for _, obj in ipairs(playermatApi.searchAroundPlayermat(matColor)) do
+      if obj.getName() == "Attachment Helper" then
+        if obj.call("getLastSyncedName") == "George Barnaby" then
+          log(obj.getQuantity())
+          return obj.getQuantity()
+        end
+      end
+    end
```

Comment:
If there is no attachment helper found for George it should probably default to something between 0 and 5 instead of 8